conbench/util.py

```diff
@@ -26,8 +26,8 @@
 
 
 def tznaive_iso8601_to_tzaware_dt(
-    input: Union[str, list[str]]
-) -> Union[datetime, list[datetime]]:
+    input: Union[str, List[str]]
+) -> Union[datetime, List[datetime]]:
```

Comment:
Cool. And yeah, that was basically an oversight, and a consequence of me not yet having done tons of typing work in Python.